### PR TITLE
Pull prometheus + eck-operator images from tigera/ to match docs pulling from quay.io/tigera

### PR DESCRIPTION
**Pull request by freecaykes**
## Description

To match the tigera-docs pulling Prometheus images and eck-operator from quay.io/tigera

## For PR author

- [x] Tests for change.
- [ ] If changing pkg/apis/, run `make gen-files`
- [ ] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [ ] Milestone set according to targeted release.
- [ ] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

File: config/enterprise_versions.yml
Comment (doublek) [inline config/enterprise_versions.yml:93]: Looks like enterprise_versions.yml is a bit mangled up (over time and not because of this PR). Noticed that there is a comment about 3rd party images but it has enterprise images in them. Can you 
1. Update the comment about third party images 
2. Reorder the images around the comment so that the comment about 3rd party images is retained but elasticsearch-operator, prometheus and alertmanager are moved under the comment and no other non 3rd party components are under it.

**Review by doublek — COMMENTED** [review]
PR looks good to me. Just added one comment about reordering images. I would also like to get input from @tmjd as well in case he has any questions/concerns.

File: config/enterprise_versions.yml
Comment (freecaykes) [reply config/enterprise_versions.yml:93]: https://github.com/tigera/operator/pull/1352/commits/88527adb4dcc3fa84f806620b4b75a42b2324bcd

File: config/enterprise_versions.yml
Comment (tmjd) [inline config/enterprise_versions.yml:82]: Could you change this comment, it really isn't very applicable anymore since we're rehosting all the images under quay.io/tigera?  Maybe something like
```
The components below are third-party images that have been retagged under quay.io/tigera so all enterprise images come from the same repository and org.
```

**Review by tmjd — APPROVED** [review]
Thanks LGTM

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
